common/src/main/java/org/apache/drill/common/map/CaseInsensitiveMap.java

```diff
@@ -55,6 +55,18 @@
   }
 
   /**
+   * Returns a new instance of {@link java.util.HashMap}, with key case-insensitivity, of expected size.
```

Comment:
The function name is `newHashMapWithExpectedSize` (following the convention of other functions in the class).